rcl/include/rcl/time.h

```diff
@@ -183,6 +200,17 @@ rcl_clock_init(
  * Passing a clock with type RCL_CLOCK_UNINITIALIZED will result in
  * RCL_RET_INVALID_ARGUMENT being returned.
  *
+ * This function is not thread-safe with any other function operating on the same
+ * clock object.
+ *
+ * <hr>
+ * Attribute          | Adherence
+ * ------------------ | -------------
+ * Allocates Memory   | Yes
```

Comment:
Where?

Reply:
Nowhere, wrong value. Fixed in 03d4a769cf9515c6101a32b27c381af9322d6706.